pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -389,5 +403,17 @@ func (plugin *cniNetworkPlugin) buildCNIRuntimeConf(podName string, podNs string
 	// Set the PodCIDR
 	rt.CapabilityArgs["ipRanges"] = [][]cniIpRange{{{Subnet: plugin.podCidr}}}
 
+	// Set dns capability args.
+	if dnsOptions, ok := options["dns"]; ok {
```

Comment:
`options` could be `nil`, right?

Reply:
Yep, ok will be false in that case

Reply:
I thought it would panic. surprised it won't.